package.json

```diff
@@ -7,7 +7,8 @@
   "jsnext:main": "es/index.js",
   "files": [
     "es",
-    "lib"
+    "lib",
+    "dist"
   ],
```

Comment:
we can add `unpkg: "dist/react-onclickoutside.min.js"`, so it would produce nicer links for the unpkg service

Reply:
Didn't know that one. 👍